starfish/experiment/builder/__init__.py

```diff
@@ -74,7 +74,7 @@ def build_image(
     collection = Collection()
     for fov_ix in range(fov_count):
         fov_images = TileSet(
-            [Coordinates.X, Coordinates.Y, Indices.Z, Indices.ROUND, Indices.CH],
+            [Indices.X, Indices.Y, Indices.Z, Indices.ROUND, Indices.CH],
```

Comment:
I thought these are meant to be coordinates? @ttung ?

Reply:
I think these are supposed to be indices.